packages/client/src/helpers.ts

```diff
@@ -47,6 +49,9 @@ export async function executionInfoFromRaw<T>(
   dataConverter: LoadedDataConverter,
   rawDataToEmbed: T
 ): Promise<Replace<WorkflowExecutionInfo, { raw: T }>> {
+  const optionalLongToNumber = (x: Long | null | undefined): number | undefined =>
+    x === undefined || x === null ? undefined : x.toNumber();
```

Comment:
Optional chaining really helps here. 
I don't think we should be using == instead of ===, many javascript programmers get confused by it, and it doesn't hurt to be a bit more explicit.

Reply:
+1 for `x == null`, we've standardized on it in the SDK codebase.

Reply:
OK, let's follow the standard